cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramNotificationSubscriberService.java

```diff
@@ -817,6 +817,10 @@ private void emitRunTimeMetric(ProgramRunId programRunId, ProgramRunStatus progr
     MetricsContext metricsContext = ProgramRunners.createProgramMetricsContext(programRunId, tags,
                                                                                metricsCollectionService);
     metricsContext.gauge(Constants.Metrics.Program.RUN_TIME_SECONDS, runTime);
+    metricsContext.gauge(Constants.Metrics.Program.EXPECTED_STOP_TIME_SECONDS,
+                         Optional.ofNullable(runRecord.getTerminateTs()).orElse(-1L));
```

Comment:
this isn't meaningful to emit, as it's just a timestamp. Graphing this will just be a straight line.